WooCommerce/WooCommerceTests/POS/ViewModels/ItemListViewModelTests.swift

```diff
@@ -23,9 +23,61 @@ final class ItemListViewModelTests: XCTestCase {
         super.tearDown()
     }
 
+    func test_itemListViewModel_when_populatePointOfSaleItems_is_called_then_items_are_populated() async {
```

Comment:
np;

I'm thinking if these tests are needed, given they do exactly the same as:
`test_itemListViewModel_when_reload_is_called_then_items_are_populated()` 

However, I'm leaving it up to you. As you mentioned in the PR, we may want to differentiate between `reload` and `popularPOSItems` later.

Reply:
Sounds good!